quickannotator/dl/training.py

```diff
@@ -98,8 +99,8 @@ def train_pred_loop(config):
     
     scaler = torch.amp.GradScaler("cuda")
 
-    print(summary(model, (3, tile_size, tile_size))) #TODO: log this
-
+    # print(summary(model, (3, tile_size, tile_size))) #TODO: log this
+    # TODO: why does the above line produce an error: RuntimeError: Input type (torch.cuda.FloatTensor) and weight type (torch.FloatTensor) should be the same
```

Comment:
strange - this would seem to mean that the function is creating a temporary tile on the GPU instead of in memory  - i don't have this error locally... if this error persists, i think you can add "device ='cpu'" to the call to force all types to be .Float and not .cuda.float